admin/assets/javascripts/global-scripts.js

```diff
@@ -377,4 +444,42 @@ $(function () {
       stickyBanner.toggle(true)
     }
   }
+
+  /**
+   * Filtering pupils by group.
+   */
+  if ($('#filterByGroup').length > 0) {
+    var groupIds = []
+    var activeGroupsIds = groupFilters.findActiveGroups()
+
+    if (activeGroupsIds) {
+      $('#filter-content').removeClass('hidden')
+      var groupIdsArr = decodeURIComponent(activeGroupsIds).split(',')
```

Comment:
@jon-shipley Initially this filter was meant to be a hide/show solution, however we lose the filter selections when the lists are re-sorted (a GET request), the only way to pass data is the URL. This can be manipulated as any of the other URL parameters passed.